### DISCUSSION TITLE
Images are not displaying in official site

### DISCUSSION THREAD

**Discussion by MaheeGamage** [saadeghi/daisyui#914] [General]
Image links are broken in the card components - https://daisyui.com/components/card/

![image](https://user-images.githubusercontent.com/26676091/176495038-84fc4821-9894-4a8f-9e86-1cc06ff7d024.png)

**Comment by saadeghi** [comment]
Yes. It's the random image API.  
I'm working on it.

**Comment by saadeghi** [comment]
Fixed.
